template/nuxt/nuxt.config.js

```diff
@@ -75,14 +75,19 @@ module.exports = {
     '@/plugins/antd-ui'
     <%_ } _%>
   ],
-  <%_ if (eslint) { _%>
   /*
-  ** Nuxt.js devModules
+  ** Nuxt.js dev-modules
   */
   devModules: [
```

Comment:
Good catch! Maybe we could remove the whole `devModules` key in the config object if neither eslint nor tailwind had been selected?